Update tls-carbon to 8.11.3 and add .gitignore

mrister opened the pull request:
Update to Node.js 8.11.3 as recommended by Node.js security team to fix known vulnerabilities.
Update lts-boron and latest images to node 6.14.3
Add .gitignore

Image built and pushed to docker hub https://hub.docker.com/r/dialonce/nodejs/tags/

mrister committed:
Update tls-carbon to 8.11.3 and add .gitignore

jkernech commented on build.sh:
```diff
@@ -8,6 +8,7 @@ docker image build --file=Dockerfile --label="Node.js LATEST -> LTS Boron" --pul
 #docker image push dialonce/nodejs:lts-boron
 
-# The Node.js LTS CARBON image, based on  Node 8.9.4
-docker image build --file=Dockerfile-LTS-Carbon --label="Node.js LTS Carbon" --pull --tag="dialonce/nodejs:lts-carbon" .
```
Did you intend to remove this line ?

mrister replied:
uh damn merges, no I'll put it back

mrister committed:
Update latest and lts-boron to node v6.14.3

ofkindness commented on Dockerfile-LTS-Carbon:
```diff
@@ -1,6 +1,6 @@
 FROM alpine:3.6
```
Why it's not latest release `3.7.0` ?

mrister replied:
Hm IDK I was not updating that, but when I initially did this Dockerfile it was a recommended/showcased version of alpine for that version of node (8), perhaps we can update and see if it is ok 🤔  I'll give it a try in a bit and update here...

mrister replied:
Unfortunately, I was unable to build an image for 3.7 successfully, IDK why it fails but perhaps it needs a newer version of node (9 or 10) so leaving it as is.

mrister committed:
Update latest and lts-Carbon to Alpine 3.7

mrister commented:
Actually managed to update to Alpine 3.7

mrister committed:
Update latest and lts-Carbon to Alpine 3.7 and NPM 6

NPM 6 is needed because on some projects there were errors with npm and they could not be used via this docker image.
 Fom more info see https://github.com/npm/npm/issues/19989

mrister commented:
Also for lts carbon I updated NPM to v 6 as without it gateway-screen would not be able to compile and work under docker.
